CHANGELOG.md

```diff
@@ -80,6 +99,27 @@ storage when upgrading from 1.5 to 1.6.  See Upgrade Notes for 1.6.x.
 * ui: Fixed and updated lease renewal picker [[GH-11256](https://github.com/hashicorp/vault/pull/11256)]
 * ui: fix issue where select-one option was not showing in secrets database role creation [[GH-11294](https://github.com/hashicorp/vault/pull/11294)]
 
+## 1.7.3
+### June 16th, 2021
+
+CHANGES:
+
+* go: Update go version to 1.15.13 [[GH-11857](https://github.com/hashicorp/vault/pull/11857)]
+
+IMPROVEMENTS:
+
+* db/cassandra: Added tls_server_name to specify server name for TLS validation [[GH-11820](https://github.com/hashicorp/vault/pull/11820)]
```

Comment:
Should this also be in the 1.8 release notes? I noticed that some of the items like the `ui` IMPROVEMENT below appears on both releases.

Reply:
I think it should be in the 1.8 section if it's in this section, unless the file only exists in the 1.7 release branch.

Reply:
This change isn't in the RC. It must have _just_ missed the cut-off point.